src/index.js

```diff
@@ -60,35 +47,20 @@ app.use((req, res, next) => {
 if (!isProduction) {
   // eslint-disable-next-line no-unused-vars
   app.use((err, req, res, next) => {
-    serverLog(err.stack);
-
     res.status(err.status || 500);
-
     res.json({
-      errors: {
-        message: err.message,
-        error: err,
-      },
+      success: false,
+      message: err.message || 'Something went wrong',
     });
   });
 }
-
 // production error handler
 // no stacktraces leaked to user
 // eslint-disable-next-line no-unused-vars
-app.use((err, req, res, next) => {
-  res.status(err.status || 500);
-  res.json({
-    errors: {
-      message: err.message,
-      error: {},
-    },
-  });
-});
 
 // finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, () => {
+const server = app.listen(process.env.PORT, () => {
   serverLog(`Listening on port ${server.address().port}`);
```

Comment:
According to TTL, we are asked to change serverLog to console.log

Reply:
Ok